src/main/webapp/user-page.html

```diff
@@ -28,30 +37,82 @@
     <script src="/js/user-page-loader.js"></script>
   </head>
   <body onload="initiateUI();">
-    <nav>
-      <ul id="navigation"></ul>
+  <!-- Website title header -->
+    <h1 class="site-heading text-center text-white d-none d-lg-block">
+      <span class="site-heading-upper text-primary mb-3">Find Great Local Food</span>
+      <span class="site-heading-lower" id="page-title">Website Name</span>
+    </h1>
+
+    <!-- Navigation bar -->
+    <nav class="navbar navbar-expand-lg navbar-dark py-lg-4" id="mainNav">
+      <div class="container">
+        <a class="navbar-brand text-uppercase text-expanded font-weight-bold d-lg-none" href="#">Website Name</a>
+        <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarResponsive" aria-controls="navbarResponsive" aria-expanded="false" aria-label="Toggle navigation">
+          <span class="navbar-toggler-icon"></span>
+        </button>
+        <div class="collapse navbar-collapse" id="navbarResponsive">
+          <ul id="navigation" class="navbar-nav mx-auto">
+          </ul>
+        </div>
+      </div>
     </nav>
-    <h1 id="page-title">User Page</h1>
-
-    <form id="message-form" action="/messages" method="POST" class="hidden">
-      Enter a new message:
-      <br/>
-      <textarea name="text" id="message-input"></textarea>
-      <br/>
-      <input type="submit" value="Submit">
-    </form>
-    <hr/>
-
-    <div id="message-container">Loading...</div>
-
-    <div id="about-me-container">Loading...</div>
-    <div id="about-me-form" class="hidden">
-      <form action="/about" method="POST">
-        <textarea name="about-me" placeholder="about me" rows=4 required></textarea>
-        <br/>
-        <input type="submit" value="Submit">
-      </form>
-    </div>
+
+    <!-- Text on background -->
+    <section class="page-section cta">
+      <div class="container">
+        <div class="row">
+          <div class="col-xl-9 mx-auto">
+            <div class="cta-inner text-center rounded">
+              <h2 class="section-heading mb-4">
+                <span class="section-heading-lower">Enter a new message:</span>
```

Comment:
Maybe "Your page" or "Write a new post"? Or even better, display the user's about me info here?